core/src/toast/pane/pane.module.css

```diff
@@ -2,7 +2,9 @@
 	display: flex;
 	align-items: center;
 	position: relative;
+}
 
+.default {
```

Comment:
Should not use the "default" name here:
1. It's not a semantic name, it's a functional name. Something like a "default color" won't tell you whether it's blue or red.
2. "default" is a keyword. It's ok to use as "style.default" but you won't be able to destruct it

It's better to use a literal naming here. Even "normal" should work. But may be "outset" like other components